src/d3a/models/strategy/load_hours.py

```diff
@@ -146,6 +139,12 @@ def event_activate(self):
         # If use_market_maker_rate is true, overwrite final_buying_rate to market maker rate
         if self.use_market_maker_rate:
             self.area_reconfigure_event(final_buying_rate=GlobalConfig.market_maker_rate)
+        for time_slot in generate_market_slot_list():
```

Comment:
I think that the validation should also be applied to the `area_reconfigure_event` method, because final rate and initial rate could also potentially change from there.